.cirrus.yml

```
@@ -144,7 +172,7 @@ macos_task:
     CIRRUS_WORKING_DIR: /tmp/zeek
 
 # FreeBSD EOL timelines: https://www.freebsd.org/security/security.html#sup
-freebsd_task:
+freebsd12_task:
   freebsd_instance:
     # FreeBSD 12 EOL: June 30, 2024
     image_family: freebsd-12-1
```

So glad to see this moving forward! 👏

One small note: while we're at it, FreeBSD 12.2 released recently:

```suggestion
    image_family: freebsd-12-2
```

---

Thanks! I wasn't sure whether that's live yet since I'm not seeing it on [their page](https://cirrus-ci.org/guide/FreeBSD/) ... I pushed the change to see what happens.

---

Looks like it works! https://cirrus-ci.com/task/4871117913456640